### PR TITLE
[diagnostics] aquire all cpu data from netdata

### LINKED ISSUE #313: [cpu_monitor] acquire all stats/data from netdata
Since netdata already collects all the data that cpu_monitor might need, and cpu_monitor already uses some stats from netdata, we should move to getting all the data entirely from netdata.

### PR DESCRIPTION
**Pull request by pgehring**
fixes #313
 ref https://github.com/mojin-robotics/cob4/issues/2245

### CODE REVIEW

**Review by fmessmer — CHANGES_REQUESTED** [review]
```
************* Module cpu_monitor
src/cob_command_tools/cob_monitoring/src/cpu_monitor.py:446: [E1120(no-value-for-parameter), CPUMonitor.check_cpu_util] No value for argument 'after' in function call
src/cob_command_tools/cob_monitoring/src/cpu_monitor.py:447: [E1120(no-value-for-parameter), CPUMonitor.check_cpu_util] No value for argument 'after' in function call
'run_pylint' returned with code '2' after 0 min 45 sec
The command ".industrial_ci/travis.sh" exited with 2.
```

File: cob_monitoring/src/netdata_tools/netdata_tools.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import requests
+import numpy
+
+def query_netdata_info():
+
+    try:
+        r = requests.get('http://127.0.0.1:19999/api/v1/info')
+    except requests.ConnectionError as ex:
+        rospy.logerr("NetData ConnectionError %r", ex)
+        return None
```
Comment (fmessmer) [inline cob_monitoring/src/netdata_tools/netdata_tools.py]: maybe we should return a tuple (data, error) - so if data is None we can write error into the diagnostics message

this should be done for all error cases in this `netdata_tools` module

File: cob_monitoring/src/cpu_monitor.py [hunk 2]
```diff
@@ -261,82 +125,37 @@ def check_core_temps(self):
...
-                                 stdout = subprocess.PIPE,
-                                 stderr = subprocess.PIPE, shell = True)
-            stdout, stderr = p.communicate()
-            retcode = p.returncode
-            try:
-                stdout = stdout.decode()  #python3
-            except (UnicodeDecodeError, AttributeError):
-                pass
+            # get max freq
+            netdata_max_cpu_freq = query_netdata_info()
```
Comment (fmessmer) [inline cob_monitoring/src/cpu_monitor.py]: is this correct?
dont we need to extract the max_cpu_freq from the info - as is done for the num_cores?

**Review by fmessmer — CHANGES_REQUESTED** [review]
I'll continue on this myself

File: cob_monitoring/src/cpu_monitor.py (hunk 2 above)
Comment (pgehring) [reply cob_monitoring/src/cpu_monitor.py]: The variable name is a bit misleading. Should be of course also `netdata_info`

**Review by fmessmer — DISMISSED** [review]
tested on my laptop
Screenshot from 2022-03-15 14-59-27

`cpu_monitor` also handles `sudo systemctl stop netdata.service` and `sudo systemctl restart netdata.service` correctly

**Pull request merged**